### ISSUE TITLE
Allow LoRA params and learning rate to be changed in total_config.yaml

### ISSUE THREAD

**Issue by niznik-dev** [niznik-dev/predicting-zygosity#114]
# Describe the Feature

As a user, I would like to be able to change LoRA and learning rate parameters in total_config.yaml so that less things need to be changed after running generate_slurm_script.py and the experience is seamless

Parameters include:
- rank (alpha will be calcuated based on this)
- lr_scheduler.num_warmup_steps
- lr_scheduler._component_ (will need to validate the component exists)

## Related Issues

(None)

# Acceptance Criteria
- [ ] generate_slurm_script.py can change the above LoRA and learning rate parameters
